src/components/navbar/PresentationSwitch.scss

```diff
@@ -0,0 +1,59 @@
+.PresentationSwitch {
+
+    margin: 0.5em;
+    background-color: transparent;
+
+    &:focus {
+        outline: 0;
+    }
+
+    &__outer {
+        vertical-align: middle;
+        width: 47px;
+        height: 24px;
+        border-radius: 999px;
+        background-color: #00B0F2;
+        -webkit-transition-duration: .4s;
+        transition-duration: .4s;
+        -webkit-transition-property: background-color, box-shadow;
+        transition-property: background-color, box-shadow;
+        cursor: pointer;
+
+        &--active {
+            width: 47px;
+            background-position: 0 0;
+            background-color: #3b89ec;
+        };
+
+        &--active {
+            background-color: #57BB00;
+        }
+    }
+
+    &__inner {
+        float: left;
+        width: 23px; height: 23px;
+        border-radius: inherit;
+        background: #ffffff;
+        -webkit-transition-timing-function: cubic-bezier(1,0,0,1);
+        transition-timing-function: cubic-bezier(1,0,0,1);
+        -webkit-transition-duration: 0.4s;
+        transition-duration: 0.4s;
+        -webkit-transition-property: transform, background-color;
+        transition-property: transform, background-color;
+        pointer-events: none;
+        margin-top: 1px;
+        margin-left: 1px;
+
+        &--active {
```

Comment:
Please use the parents `&--active` class instead of giving this class to outer and inner too. 
It makes the javascript code cleaner and removes duplication.